import-service/src/functions/importFileParser/handler.ts

```diff
@@ -17,8 +21,15 @@ const importFileParser = async (event) => {
                 return new Promise<void>((resolve, reject) => {
                     stream
                         .pipe(csv())
-                        .on('data', (data) => {
-                            console.log(item.s3.object.key, ': ', data);
+                        .on('data', async (data) => {
```

Comment:
if I'm not mistaken, the "on" method does not support asynchronous callbacks and data will be lost if the process closes before all messages have been sent. You have two ways to handle this, the first one is to create an array of promises and use Promise.all, and the second one is to pause the stream, await promise, and resume the stream then